proj/src/main/webapp/chat.js

```diff
@@ -729,8 +777,11 @@ function addUserInfoToDom(userObj) {
     const profile = document.getElementById('user-profile');
     if (userObj.uid !== currentUID) {
```

Comment:
This one too if possible, but I think maybe we can save it for a later PR since it isn't new.